lib/katakata_irb/types.rb

```diff
@@ -186,12 +186,16 @@ def initialize(klass, params = {})
       @params = params
     end
     def transform() = yield(self)
-    def methods() = @klass.instance_methods
-    def all_methods() = @klass.instance_methods | @klass.private_instance_methods
+    def methods() = rbs_methods.select { _2.public? }.keys | @klass.instance_methods
+    def all_methods() = rbs_methods.keys | @klass.instance_methods | @klass.private_instance_methods
     def constants() = []
     def types() = [self]
     def nillable?() = (@klass == NilClass)
     def nonnillable() = self
+    def rbs_methods
+      type_name = RBS::TypeName(KatakataIrb::Types.class_name_of(@klass)).absolute!
```

Comment:
`KatakataIrb::Types.class_name_of` might return `nil`. (example: anonymous class, singleton class)
Could you add a nil check to name like the code below?

https://github.com/tompng/katakata_irb/blob/46849d800d551e089b934ed4733bc40f15bb7c14/lib/katakata_irb/types.rb#L30-L32